config/lava/chromeos/cros-baseline.jinja2

```diff
@@ -0,0 +1,30 @@
+{% extends base_template|default('chromeos/cros-boot.jinja2') %}
+{% block actions %}
+{{ super() }}
+- test:
+    namespace: chromeos
+    timeout:
+      minutes: 5
+    docker:
+      image: {{ docker_image }}
+      wait:
+        device: true
+    results:
+      location: /home/cros/lava
+    definitions:
+    - repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: cros-baseline
+        run:
+          steps:
+            - cd /home/cros
+            - while ! ping -c 1 -w 1 $(lava-target-ip); do sleep 1; done
+            - >-
+              KERNELCI_LAVA=y
+              cmdwrapper="ssh -i /home/cros/.ssh/id_rsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null root@$(lava-target-ip)"
```

Comment:
So how about adding the timeout option, to be sure it's long enough?

Reply:
If host is pingable, it should not fail if ssh up too, and will be long enough by default, as i recall it is based on default SYN timeout + number of retries.
We might face case when ssh is not up yet, and connection refused, but as i recall ssh don't have mechanism to retry on such case.